### PR TITLE
Add info about the `--temp-path` option

### PR DESCRIPTION
**Pull request by NataliaIvakina**

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1891.surge.sh

Updated pages:
- [Aggregate a database backup chain](https://neo4j-docs-operations-1891.surge.sh/operations-manual/5/backup-restore/aggregate)
- [Copy a database store](https://neo4j-docs-operations-1891.surge.sh/operations-manual/5/backup-restore/copy-database)
- [Back up an online database](https://neo4j-docs-operations-1891.surge.sh/operations-manual/5/backup-restore/online-backup)
- [Restore a database backup](https://neo4j-docs-operations-1891.surge.sh/operations-manual/5/backup-restore/restore-backup)

File: modules/ROOT/pages/backup-restore/restore-backup.adoc [hunk 1]
```diff
@@ -131,7 +131,12 @@ For more information, see <<restore-cloud-storage>>.
 
 [NOTE]
 ====
-The `--temp-path` option is available starting from Neo4j 5.24.
+Neo4j 5.24 introduces the `--temp-path` option to address potential issues related to disk space when performing backup-related commands, especially when cloud storage is involved. 
+
+If `--temp-path` is not set, a temporary directory is created inside the current working directory for Neo4j.
```
Comment (Frannie-Ludmilla) [inline modules/ROOT/pages/backup-restore/restore-backup.adoc:136]: Hi! I had to double check: in this specific case, first it goes via the path defined in `--from-path` and then the current working directory

File: modules/ROOT/pages/backup-restore/aggregate.adoc [hunk 2]
```diff
@@ -106,7 +106,12 @@ For more information, see <<aggregate-backup-cloud-storage>>.
 
 [NOTE]
 ====
-The `--temp-path` option is available starting from Neo4j 5.24.
+Neo4j 5.24 introduces the `--temp-path` option to address potential issues related to disk space when performing backup-related commands, especially when cloud storage is involved. 
+
+If `--temp-path` is not set, a temporary directory is created inside the current working directory for Neo4j.
```
Comment (Frannie-Ludmilla) [inline modules/ROOT/pages/backup-restore/aggregate.adoc:111]: Hi! I had to double check: in this specific case, first it goes via the path defined in --from-path and then the current working directory

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1891.surge.sh

Updated pages:
- [Aggregate a database backup chain](https://neo4j-docs-operations-1891.surge.sh/operations-manual/5/backup-restore/aggregate)
- [Copy a database store](https://neo4j-docs-operations-1891.surge.sh/operations-manual/5/backup-restore/copy-database)
- [Back up an online database](https://neo4j-docs-operations-1891.surge.sh/operations-manual/5/backup-restore/online-backup)
- [Restore a database backup](https://neo4j-docs-operations-1891.surge.sh/operations-manual/5/backup-restore/restore-backup)

File: modules/ROOT/pages/backup-restore/aggregate.adoc (hunk 2 above)
Comment (NataliaIvakina) [reply modules/ROOT/pages/backup-restore/aggregate.adoc]: Updated

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1891.surge.sh

Updated pages:
- [Aggregate a database backup chain](https://neo4j-docs-operations-1891.surge.sh/operations-manual/5/backup-restore/aggregate)
- [Copy a database store](https://neo4j-docs-operations-1891.surge.sh/operations-manual/5/backup-restore/copy-database)
- [Back up an online database](https://neo4j-docs-operations-1891.surge.sh/operations-manual/5/backup-restore/online-backup)
- [Restore a database backup](https://neo4j-docs-operations-1891.surge.sh/operations-manual/5/backup-restore/restore-backup)

**Review by Frannie-Ludmilla — APPROVED** [review]

**Pull request merged**